### PR TITLE
Template interpreter

### PR DESCRIPTION
**Pull request by LeonardBesseau**
Feature branch to implement a template interpreter with handlesbar

### CODE REVIEW

**Comment by LeonardBesseau** [comment]
Close #55

**Comment by LeonardBesseau** [comment]
Made ExtractAll in PageParser which extracts metaData and content at the same time. Do we want to keep the others ?

File: src/main/java/ch/heig_vd/app/Main.java [hunk 1]
```diff
@@ -6,14 +6,23 @@
...
 
+import java.util.Properties;
 import java.util.concurrent.Callable;
 
 import static java.lang.System.exit;
 
 // Main command info
+
+/**
+ * @author Besseau Léonard
```
Comment (MazieroMarco) [inline src/main/java/ch/heig_vd/app/Main.java:17]: So much humility wow

Comment (LeonardBesseau) [reply src/main/java/ch/heig_vd/app/Main.java:17]: image

**Review by MazieroMarco — APPROVED** [review]
All reviewed, ok to merge

**Pull request merged**